batterup/src/App.test.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+import App from './App';
+import Display from './components/Display/Display';
+import Dashboard from './components/Dashboard/Dashboard';
+
+// automatically unmount and cleanup DOM after the test is finished.
+// beforeEach(cleanup);
+
+describe('The Batter Up React App', () => {
+  it('renders without crashing', () => {
+    render(<App />);
+  });
+
+  describe('the display component', () => {
+    it('displays the correct number of strikes when the "Strike" button is clicked.', () => {
+      const dashboard = render(<Dashboard />);
+      const strikeBtn = dashboard.getByText(/strike/i);
+      fireEvent.click(strikeBtn);
+
```

Comment:
I like how you're grouping both the presentation and functionality testing together here Jesse 🔥 This pattern starts to read like the steps a user would take when (in this case) they're interacting with the strike button of the app.